Formula/liquidprompt.rb

```diff
@@ -17,13 +17,6 @@ def caveats; <<-EOS.undent
       if [ -f #{HOMEBREW_PREFIX}/share/liquidprompt ]; then
         . #{HOMEBREW_PREFIX}/share/liquidprompt
       fi
-
-    If you'd like to reconfigure options, you may do so in ~/.liquidpromptrc.
-    A sample file you may copy and modify has been installed to
-      #{HOMEBREW_PREFIX}/share/liquidpromptrc-dist
```

Comment:
You might potentially want to keep this part as well, since it points the where Homebrew installed the sample `.liquidpromptrc`, but it isn't nearly as important as the first section of the caveat, which is required for basic functionality.